src/components/widgets/DefaultWidget.tsx

```diff
@@ -31,11 +31,17 @@ export const DefaultWidget = ({ Icon, count, onPress, text }: Props) => {
   );
 };
 
+/* eslint-disable react-native/no-unused-styles */
+/* this works properly, we do not want that warning */
```

Comment:
we do not need this anymore without function, right?

Reply:
Yes, you're right, we don't need it. 👍